events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -176,11 +231,22 @@
         category = null;
         difficulty = null;
     }
-  
+    
     function findTargets() {
         Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element) {
             var name = Entities.getEntityProperties(element, ['name']).name;
             if (name.indexOf("Trivia") !== -1) {
+                var hasServerScript = Entities.getEntityProperties(element, ['serverScripts']).serverScripts;
+                if (hasServerScript !== "") {
```

Comment:
`hasServerScript` feels like a misnomer to me. perhaps something like `serverScriptURL` ?